app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1 +1,31 @@
 /* Placeholder for backend dummy app */
+SpreePayuIntegration = {
+  updateSaveAndContinueVisibility: function() {
```

Comment:
this method needs to be called on init, and you cannot just call `toggle` because you need to know which payment method is currently selected